integration-tests/test-suite/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Validates fluent bit end-to-end with several configurations",
   "scripts": {
-    "test": "jest --verbose fluent-bit-tests.test.js"
+    "test": "jest --verbose --forceExit fluent-bit-tests.test.js"
```

Comment:
What is the `forceExit` flag for? Would it make sense to add a comment here as to why we are using it?

Reply:
I needed this at some point otherwise the machines were left in an inconsistent state. I think we can remove it though. I'll follow up in the next PR.

Reply:
Removed!